parsl/tests/test_bash_apps/test_basic.py

```diff
@@ -24,6 +24,7 @@ def foo(x, y, z=10, stdout=None, label=None):
     return f"echo {x} {y} {z}"
 
 
+@pytest.mark.shared_fs
```

Comment:
maybe all these shared_fs / staging_required markers should be merged as a separate PR? They're not specific to this test case (eg. they came I think originally from the kubernetes testing draft PR #3482 and that work would be slightly advanced by these being merged to master, independent of the rest of this PR being merged or not)

Reply:
Fair enough. I left this here because those changes wouldn't kick in without tests that exclude based on the marker. I've split out these changes over on this PR -> https://github.com/Parsl/parsl/pull/3669